contrib/jenkins_tests/build.sh

```diff
@@ -13,19 +13,20 @@ cd ${build_dir}
 # Set symbolic links to default build and install
 ln -s "${build_dir}/0/install" "${install_dir}"
 
-build_list="\
-debug:--enable-opt-log=no --enable-debug \
-nginx-off:--enable-nginx=no \
-envoy-on:--enable-nginx=yes \
-default: "
-
+declare -A build_list
+build_list['debug']="--enable-opt-log=no --enable-debug"
+build_list['nginx-off']="--enable-nginx=no"
+build_list['envoy-on']="--enable-nginx=yes"
+build_list['static-on']="--enable-static --disable-shared"
+build_list['default']=""
```

Comment:
I'd also prefer to see maximum build (nginx, envoy) and --enable-opt-log=high. "high" value is not covered by these builds and is a source of warnings that can be easily missed until the release build.

Not necessary in current PR.

Reply:
This change is designed to fix the bug where build_list with multiple options was broken.
In addition, I added the test for static build to CI.
Is there anything else you suggest adding here?